Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -1692,7 +1691,19 @@ open class NavigationMapView: UIView {
     // MARK: Map Rendering and Observing
     
     var routes: [Route]?
-    var continuousAlternatives: [AlternativeRoute]?
+    var continuousAlternatives: [AlternativeRoute]? {
+        didSet {
+            alternativesRouteLineDeviationOffsets = continuousAlternatives?.map {
+                guard let coordinates = $0.indexedRouteResponse.currentRoute?.shape?.coordinates,
+                      let projectedOffset = calculateGranularDistanceOffset(coordinates,
```

Comment:
These calculations are specific to Web Mercator projection. If the map SDK’s globe view is enabled, will the annotations be placed in a reasonable location? Perhaps we should disable the annotations when globe view is enabled or track a fix separately.

Reply:
Globe projection also displays annotations and route line vanished portion correctly.